CHANGELOG.md

```diff
@@ -19,6 +19,10 @@ The types of changes are:
 
 ## [Unreleased](https://github.com/ethyca/fidesops/compare/1.6.1...main)
 
+### Added
+* Adds SaaS connection type to SaaS yaml config [748](https://github.com/ethyca/fidesops/pull/748)
```

Comment:
Are you sure this is updated with main? the changelog there has more items in unreleased?

https://github.com/ethyca/fidesops/blob/c5c30840d07d2c2c4b999bdf241044690c63d092/CHANGELOG.md

Reply:
ok, this looks better now